builder/azure/arm/builder.go

```diff
@@ -91,6 +95,18 @@ func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packe
 		return nil, err
 	}
 
+	_, _, err = p.ParseUnverified(spnCloud.OAuthToken(), claims)
+
+	if err != nil {
+		return nil, err
+	}
+	b.config.ObjectID = claims["oid"].(string)
+
+	if b.config.ObjectID == "" && b.config.OSType != constants.Target_Linux {
+		ui.Error("\n Got empty Object ID in the OAuth token , we need this for Key vault Access, bailing")
```

Comment:
AFAIK, it's not a common practice in this code base to start errors with `\n`, I assume this is to make the error stand out more?

Reply:
We don't say "bailing" in Packer. 😁 

The correct thing to do is to return an error. The upper layers will properly report it to the user.

```go
return nil, fmt.Errorf("could not determined the ObjectID for the user %s, which is required for Windows builds", "<fill in user name")
```

Please feel free to change the error as necessary.  I believe the error you should be the continuation of a sentence, so lower case and not a new line.  It will already be colored red (if supported) to make the error stand out.